manual.md

```diff
@@ -74,9 +74,11 @@ If a proposal does not pass _Formal Review_, it can be resubmitted in the next g
 Submitting a proposal on-chain is the final step in the governance process. An RGP can be submitted to on-chain governance by anyone who has an amount of $RAD greater than or equal to **1% of the total $RAD supply** delegated to their address (i.e. ≥ 1M $RAD). If a proposal author does not meet this requirement, they should reach out to the Governance Committee as soon as possible. 
 
 For Submission, proposal authors are required to: 
-- Post an updated version of their proposal on the forum that incorporates feedback from the _Discussion_ and _Formal Review_ phases.
+- Post an updated version of their proposal on the forum that incorporates feedback from the _Discussion_ and _Formal Review_ phases. This version of the proposal should be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum.
+- Create an on-chain proposal using Tally or Boardroom.
```

Comment:
Should we link to a "how to" guide? I'm sure Tally has one...

Reply:
Good idea - how about this one? https://docs.tally.xyz/knowledge-base/proposals/creating-proposals